core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -420,6 +430,23 @@ public static MetricsReporter loadMetricsReporter(Map<String, String> properties
       return LoggingMetricsReporter.instance();
     }
 
+    List<String> reporters = COMMA.splitToList(impl);
+    if (reporters.size() == 1) {
+      return loadSingleMetricsReporter(properties, impl);
+    }
+
+    LOG.info("Loading multiple MetricsReporter implementations: {}", impl);
```

Comment:
let's log it for both cases (single or plural)

Reply:
logging for the single case already happens in `loadSingleMetricsReporter()`